bot.ts

```diff
@@ -36,90 +57,158 @@ const commands: SlashCommandPartial[] = [
 		options: [
 			{
 				name: "game",
-				description: "A game's abbreviation",
+				description: "A game's URL",
 				type: ApplicationCommandOptionType.STRING,
 				autocomplete: true,
 				required: true,
 			},
+			mentionUserOption,
 		],
 	},
 	{
 		name: "series-community",
-		description: "Get a link to a series' community",
+		description: "Get a link to a Series' community",
 		options: [
 			{
 				name: "series",
-				description: "A series' abbreviation",
+				description: "A Series' URL",
 				type: ApplicationCommandOptionType.STRING,
 				autocomplete: true,
 				required: true,
 			},
+			mentionUserOption,
 		],
 	},
+	{
+		name: "github",
+		description: "Get a link to the bot's source code on GitHub",
+	}
 ];
 
 class SpeedrunCom extends ApplicationCommandsModule {
 	@slash("game-community")
-	async gameCommunity(t: ApplicationCommandInteraction) {
-		await t.defer();
+	async gameCommunity(i: ApplicationCommandInteraction) {
+		await i.defer();
 		const res = await fetch(
-			`https://www.speedrun.com/api/v1/games/${t.option("game")}`,
+			`https://www.speedrun.com/api/v1/games/${i.option("game")}`,
 		);
-		const game = (await res.json()).data as SearchResult;
-
-		if (game) {
-			await t.reply({
-				content: game.discord
-					? `Here's the invite to \`${game.names.international}\`: ${game.discord}`
-					: `Couldn't find any invite for \`${game.names.international}\`, here's a link to their forums: ${game.weblink}/forum`,
-			});
-		} else {
-			await t.reply({
-				content: `\`${t.option("game")}\` game not found`,
-			});
+		const game: v1SearchResult | undefined = (await res.json()).data;
+		const userMention = i.option("mention");
+
+		if (!game) {
+			await i.reply(
+				userMention
+					? `${userMention}, the \`${i.option("game")}\` game wasn't found`
+					: `\`${i.option("game")}\` game not found`
```

Comment:
The message for both with mention or no mention should be the same, with the added mention if available.

So:
- `@AnInternetTroll the mcee game wasn't found`
- `the mcee game wasn't found`

(probably with "The" capitalized as well)

You can judge for yourself if a comma or colon should be put there, I'm fine with it either way